### ISSUE TITLE
Series Tutorials

### ISSUE THREAD

**Issue by 315574925** [odpi/egeria#7826]
### Please describe the new behavior that that will improve Egeria

Is there a series of video tutorials (from building the environment to developing and deploying) uploaded to YouTube（YouTube Not the latest and practical）

**Comment by mandy-chessell** [comment]
The tutorials and dojos are found on our website here https://egeria-project.org/education/.  There are different choices on the style/depth of tutorial to choose from.  They are web pages rather than videos.  We have been discussing whether to record some videos or not.  I have found that they can go too fast, and it is difficult to keep up with it when following along for the first time.

There is currently work underway to simplify and improve Egeria's up and running experience.  The current material refers to the original build/deployment approach and should still work (fingers crossed :).  Let me know if you have problems and I will help you.

**Comment by 315574925** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by 315574925** [comment]
I'll give it a try first. If I encounter any problems, I'll consult. Thank you very much. If you have time, it would be very intuitive to suggest recording a video.

**Comment by 315574925** [comment]
![image](https://github.com/odpi/egeria/assets/34641313/a729615a-3488-4442-ad1d-2345e5f3cd3f)
![image](https://github.com/odpi/egeria/assets/34641313/703d2f83-edb2-4152-8ad5-9a6a1a66cca5)

Path to Jar Choose 1 or 2?

Can you briefly explain the role of the files under the egeria-4.3-SNAPSHOT distribution directory。

Thank you

**Comment by mandy-chessell** [comment]
Well spotted ... this is an area we are working on and the documentation is not right for the new structure.  You need the platform/lib/omag-server-platform-4.3-SNAPSHOT.jar.

This jar file was under server/lib in previous releases.  Now server/lib contains an experimental runtime that supports only a single server.

I will update the documentation.

**Comment by 315574925** [comment]
![image](https://github.com/odpi/egeria/assets/34641313/9008d386-7cc1-4783-b4b1-b06356847a4a)
![image](https://github.com/odpi/egeria/assets/34641313/8b8f03b8-96ec-4e66-9cc1-c1c86b31de89)
![image](https://github.com/odpi/egeria/assets/34641313/112aa9e4-5086-4160-9d6c-d340e79082fe)
![image](https://github.com/odpi/egeria/assets/34641313/d4f6d926-84e5-4a6e-a678-a6133ae49cab)
run OMAGServerPlatform、egeria-dev-projects-main/egeria-config-utility/ServerConfig.java、egeria-dev-projects-main/egeria-platform-report/EgeriaPlatformReport.java, Different from the expected results of the tutorial.

**Comment by 315574925** [comment]
@mandy-chessell Hello, can you help me answer this？

**Comment by mandy-chessell** [comment]
I am struggling to see more than superficial differences between the website and the levels of IntelliJ / egeria software.   

This is what I see:
* The messages output during the platform start up have been improved.  There is more information about the working directory and trust stores.  However, I can not see any error.  The hightlighted message showing the port has not changed.
* ServerConfig is listing an additional option for the infrastructure connector.  Again I can not see any errors.
* EgeriaPlatformReport shows the additional components that have been implemented since the screenshot was taken.  I can't see any error here too.

In summary, I just see the enhancements that have been made to Egeria since the screenshots were taken but no errors.
What are the running errors you are seeing?

If the problem is the slight differences in the screen captures and the runtime, I can add a note to the dojo saying that the exact output may vary from release to release.

**Comment by 315574925** [comment]
@mandy-chessell 
Hello, you are right, but it is different from the expected results of the tutorial. When entering 'create metadata store mds1' in the 'ServerConfig' window and running 'Egeria Platform Report', there is no output of 'mds1' related information。
I am using the latest version 4.2 on the official website。
Is there a version 4.2 tutorial available。
Thank you。
![image](https://github.com/odpi/egeria/assets/34641313/a8e9571f-fa3e-4379-a7c8-79a44c7bcf15)
![image](https://github.com/odpi/egeria/assets/34641313/d5d537bc-8595-4a36-8555-2b40c54a9d7a)
![image](https://github.com/odpi/egeria/assets/34641313/a98d9b11-6db8-4af0-a72c-3b078e45dc55)
